### PR TITLE
Trailer & Multi video per page support [INDEV]

### PR DESCRIPTION
**Pull request by OfficialNoob**
Resolves https://github.com/ajayyy/SponsorBlock/issues/233, https://github.com/ajayyy/SponsorBlock/issues/150 and https://github.com/ajayyy/SponsorBlock/issues/58

### CODE REVIEW

**Comment by OfficialNoob** [comment]
This code has not been test much.

**Comment by ajayyy** [comment]
Note: This would also have to figure out when to call videoID change, since that is normally only called when a URL change occurs.

**Comment by ajayyy** [comment]
Maybe there is a way to listen for a video source change?

**Comment by OfficialNoob** [comment]
Maybe

**Comment by OfficialNoob** [comment]
@ajayyy there is a problem. do not know what

File: content.js [hunk 1]
```diff
@@ -67,101 +60,104 @@ var popupInitialised = false;
...
+                break;
+            case "changeDeleteButtonPlayerControlsVisibility":
+                SB.config.hideDeleteButtonPlayerControls = request.value;
+                updateVisibilityOfPlayerControlsButton();
 
-if (!SB.configListeners.includes(contentConfigUpdateListener)) {
-    SB.configListeners.push(contentConfigUpdateListener);
+                break;
+            case "trackViewCount":
+                SB.config.trackViewCount = request.value;
```
Comment (ajayyy) [inline content.js]: It seems you have added back some old code

**Comment by ajayyy** [comment]
So, are you basically trying to convert everything to support more than one video at a time?

**Comment by OfficialNoob** [comment]
yes

**Comment by OfficialNoob** [comment]
Failed

**Pull request closed without merging**